src/hash-table.js

```diff
@@ -1,12 +1,43 @@
 /* eslint-disable no-unused-vars */
-const { LimitedArray, getIndexBelowMax } = require('./hash-table-helpers');
-
 class HashTable {
   constructor() {
     this.limit = 8;
     this.storage = new LimitedArray(this.limit);
     // Do not modify anything inside of the constructor
   }
+  insert(key, value) {
+    const index = getIndexBelowMax(key, this.limit);
+    const pair = [key, value];
+    const bucket = this.storage.get(index);
+    if (!bucket) this.storage.set(index, [pair]);
+    else {
+      for (let i = 0; i < bucket.length; i++) {
+        if (bucket[i][0] === key) bucket[i][1] = value;
+      }
+      bucket.push(pair);
+    }
+    if (this.storage.length >= this.limit * 0.75) {
```

Comment:
Seems like you aren't reprocessing your current keys, as your hash will more than likely give different values below limit.